contracts/ChargedParticles.sol

```diff
@@ -117,10 +128,29 @@ contract ChargedParticles is
     return address(_chargedSettings);
   }
 
+  function getManagersAddress() external view virtual override returns (address managersAddress) {
+    return address(_chargedManagers);
+  }
+
   function onERC721Received(address, address, uint256, bytes calldata) external virtual override returns (bytes4) {
     return IERC721ReceiverUpgradeable(0).onERC721Received.selector;
   }
 
+  function onERC1155Received(address, address, uint256, uint256, bytes calldata) external virtual override returns (bytes4) {
+    return IERC1155ReceiverUpgradeable(0).onERC1155Received.selector;
```

Comment:
general / solidity

why the (0)? is IERC1155ReceiverUpgradeable an array under the hood?